app/src/main/java/com/kickstarter/ui/activities/ViewPledgeActivity.java

```diff
@@ -68,124 +73,140 @@ public void onCreate(final @Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.view_pledge_layout);
     ButterKnife.bind(this);
+
     final RewardsItemAdapter rewardsItemAdapter = new RewardsItemAdapter();
-    rewardsItemRecyclerView.setAdapter(rewardsItemAdapter);
+    this.rewardsItemRecyclerView.setAdapter(rewardsItemAdapter);
     final LinearLayoutManager layoutManager = new LinearLayoutManager(this);
-    rewardsItemRecyclerView.setLayoutManager(layoutManager);
+    this.rewardsItemRecyclerView.setLayoutManager(layoutManager);
 
     final Environment environment = environment();
-    ksString = environment.ksString();
-
-    RxView.clicks(projectContextView)
-      .compose(bindToLifecycle())
-      .observeOn(mainThread())
-      .subscribe(__ -> viewModel.inputs.projectClicked());
+    this.ksString = environment.ksString();
 
-    viewModel.outputs.backerNameTextViewText()
+    this.viewModel.outputs.backerNameTextViewText()
       .compose(bindToLifecycle())
-      .observeOn(mainThread())
-      .subscribe(backerNameTextView::setText);
+      .compose(observeForUI())
```

Comment:
`Transformers.observeForUI` is an transformer we've slowly been moving towards for rendering our UI components more efficiently..see https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/rx/transformers/Transformers.java#L184 for more documentation :neckbeard: